### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kemirdin**
I have read this article and I found it very nice for good understanding JavaScript especially for beginners.

### CODE REVIEW

**Comment by Kemirdin** [comment]
hi, I just would like to contribute some article and share with HYF, I don't know it didn't work.

File: Week1/README.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ Only watch the below chapters:
 
 - Helpful resource: http://jsbooks.revolunet.com/ (here you can find tons of free JavaScript books online)
 
+-You are here: Home / Learn To Program / The 50 Best Websites to Learn JavaScript
```
Comment (mkruijt) [inline Week1/README.md]: hi @Kemirdin thanks for your contribution. Can you delete this line? I don't get why it says "You are here: Home / Learn To Program / " besides that i think it is a good resource to add. I you modify it I will merge your PR

**Review by mkruijt — CHANGES_REQUESTED** [review]
Hi @Kemirdin cool that you are contributing, and yes you always have to make a pr! (the master branch is protected)

**Review by Kemirdin — COMMENTED** [review]
it's my pleasure share and contribute.thanks for the merge, I have deleted that line.

**Comment by Kemirdin** [comment]
it's my pleasure share and contribute.thanks for the merge, I have deleted that line.

**Review by mkruijt — APPROVED** [review]
Thank you!

**Pull request merged**